torch/fx/experimental/GraphManipulation.py

```diff
@@ -24,3 +27,62 @@ def replace_target_nodes_with(
         else:
             val_map[node] = new_graph.node_copy(node, lambda n : val_map[n])
     fx_module.graph = new_graph
+
+class NodeSize(NamedTuple):
+    output_size: int
+    total_size: int
+
+def get_size_of_all_nodes(fx_module: GraphModule, args: List[torch.Tensor]) -> None:
+    """Given a fx graph module, update each node with its total size (weights + bias + output)
+    and its output_size(output). For a non-module node, the total size is the output size.
+    return total size"""
+    # Mark shape and dtype for each node (node.shape and node.dtype)
+    ShapeProp(fx_module).propagate(*args)
+    # Calculate the total size of the whole fx graph
+    total_size_of_graph = 0.0
+    for node in fx_module.graph.nodes:
+        if node.name == 'output':
+            break
+        node.NodeSize = get_size_of_node(fx_module, node)
```

Comment:
nit: Python convention for attribute names is snake_case, so this should probably be `node.node_size = ...`. Or alternatively, `node.size_bytes = ...`